hardhat.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
     },
     namedAccounts: {
         deployer: 0,
+        acc1: 1,
```

Comment:
Just on these, idk if having names accounts is worth? You can also just pull in the accounts directly and index into the accounts array to access accounts